src/main/java/uk/gov/hmcts/reform/juddata/validators/AppointmentValidator.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.juddata.validators;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Set;
+
+public class AppointmentValidator implements ConstraintValidator<Appointment, String> {
+
+    private static final String VALUE1 = "CRTS TRIB - RS Admin User";
+    private static final String VALUE2 = "MAGS - AC Admin User";
+    private static final String VALUE3 = "Person on a List";
```

Comment:
move these in constant class

Reply:
Since these constants don't need to be shared with other classes, it is better to keep these constants local to the class where these are used.

Reply:
If the constants are strongly tied to an existing class or interface, you should add them to the class or interface. - Joshua Bloch